loader/app/main.py

```diff
@@ -60,7 +61,8 @@ def main():
 
     data = extract(data_dir)
     policies = transform(data)
-    load(policies)
+    for db in get_db():
+        load(db, policies)
```

Comment:
This for loop is a bit mysterious

Reply:
Yes, I copied the `get_db` from the backend, which is used by FastAPI `Dependency`. Perhaps we can use a context manager and have `with get_db() as session:` or something.

```
def get_db():
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()
```